igcollect/switch.py

```diff
@@ -328,6 +355,23 @@ def ports_stats(prefix, snmp, ports, model):
                 print(template.format(port_name, counter, data))
 
 
+def dom_stats(prefix, snmp, ports, model):
+    """
+    Print graphite-compatible Digital Optical Monitoring stats for each port
+    of the switch
+    """
+
+    for metric, oid in DOM_OIDS[model].items():
+        # SNMP is slow, so we want need to get current time for each counter.
```

Comment:
`.., so we want need ...`, want or need?

Reply:
That's fixed.